### ISSUE TITLE
1.1.1 findPrebidCreativeSize function does not work with Compile error

### ISSUE THREAD

**Issue by yoalex5** [prebid/prebid-mobile-android#116]
## Type of issue
bug

## Description
If add PrebidSDK as Maven artifact AndroidStudio does not see `Util.CreativeSizeCompletionHandler` and `Util.CreativeSize`

<img width="612" alt="Screen Shot 2019-06-27 at 1 28 58 PM" src="https://user-images.githubusercontent.com/8141931/60259308-a8490f00-98df-11e9-82e9-713f74ddf8fb.png">

<img width="279" alt="Screen Shot 2019-06-27 at 1 31 21 PM" src="https://user-images.githubusercontent.com/8141931/60259411-df1f2500-98df-11e9-8894-e7585ef8cd23.png">

## Steps to reproduce
1. Create a new Android project
2. add PrebidSDK Maven artifact (build.gradle(module) -> dependencies scope -> `implementation 'org.prebid:prebid-mobile-sdk:1.1.1'`)
3. Try to use utility function `Util.findPrebidCreativeSize()`

### Expected results
Utility classes(`CreativeSizeCompletionHandler`, `CreativeSize`) should be available from a client side

### Actual results
Static nested classes are not open to use

## Platform details
PrebidSDK v1.1.1

## Other information
 I think that the issue is in proguard

**Comment by bszekely1** [comment]
Resolved with 1.1.2 release

**Comment by yoalex5** [comment]
The API

```
                Util.findPrebidCreativeSize(dfpAdView, new Util.CreativeSizeCompletionHandler() {
                    @Override
                    public void onSize(@Nullable Util.CreativeSize size) {
                        if (size != null) {
                            dfpAdView.setAdSizes(new AdSize(size.getWidth(), size.getHeight()));
                        }
                    }
                });
```
